bundles/org.openhab.binding.lcn/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,637 @@
+			<channel-group typeId="leds" id="led"/>
+			<channel-group typeId="relays" id="relay"/>
+			<channel-group typeId="rollershutterrelays" id="rollershutterrelay"/>
+			<channel-group typeId="logics" id="logic"/>
+			<channel-group typeId="binarysensors" id="binarysensor"/>
+			<channel-group typeId="variables" id="variable"/>
+			<channel-group typeId="rvarsetpoints" id="rvarsetpoint"/>
+			<channel-group typeId="rvarlocks" id="rvarlock"/>
+			<channel-group typeId="thresholdregisters1" id="thresholdregister1"/>
+			<channel-group typeId="thresholdregisters2" id="thresholdregister2"/>
+			<channel-group typeId="thresholdregisters3" id="thresholdregister3"/>
+			<channel-group typeId="thresholdregisters4" id="thresholdregister4"/>
+			<channel-group typeId="s0inputs" id="s0input"/>
+			<channel-group typeId="keyslocktablea" id="keylocktablea"/>
+			<channel-group typeId="keyslocktableb" id="keylocktableb"/>
+			<channel-group typeId="keyslocktablec" id="keylocktablec"/>
+			<channel-group typeId="keyslocktabled" id="keylocktabled"/>
+			<channel-group typeId="codes" id="code"/>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:lcn:module"/>
+	</thing-type>
+
+	<thing-type id="group">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="pckGateway"/>
+		</supported-bridge-type-refs>
+
+		<label>LCN Group</label>
+		<description>An LCN group with multiple modules, configured in LCN-PRO</description>
+
+		<channel-groups>
+			<channel-group typeId="outputs" id="output"/>
+			<channel-group typeId="rollershutteroutputs" id="rollershutteroutput"/>
+			<channel-group typeId="leds" id="led"/>
+			<channel-group typeId="relays" id="relay"/>
+			<channel-group typeId="rollershutterrelays" id="rollershutterrelay"/>
+			<channel-group typeId="logics" id="logic"/>
+			<channel-group typeId="binarysensors" id="binarysensor"/>
+			<channel-group typeId="variables" id="variable"/>
+			<channel-group typeId="rvarsetpoints" id="rvarsetpoint"/>
+			<channel-group typeId="rvarlocks" id="rvarlock"/>
+			<channel-group typeId="thresholdregisters1" id="thresholdregister1"/>
+			<channel-group typeId="thresholdregisters2" id="thresholdregister2"/>
+			<channel-group typeId="thresholdregisters3" id="thresholdregister3"/>
+			<channel-group typeId="thresholdregisters4" id="thresholdregister4"/>
+			<channel-group typeId="s0inputs" id="s0input"/>
+			<channel-group typeId="keyslocktablea" id="keylocktablea"/>
+			<channel-group typeId="keyslocktableb" id="keylocktableb"/>
+			<channel-group typeId="keyslocktablec" id="keylocktablec"/>
+			<channel-group typeId="keyslocktabled" id="keylocktabled"/>
+			<channel-group typeId="codes" id="code"/>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:lcn:group"/>
+	</thing-type>
+
+	<channel-type id="output" advanced="true">
+		<item-type>Dimmer</item-type>
+		<label>Output</label>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Comment:
Please note that your binding is expected to quickly provide state updates after receiving commands, if you go for a veto here.
Is that really the case for all the channels where you have put "veto"?